### ISSUE TITLE
Could Not Parse Website Exception

### ISSUE THREAD

**Issue by quantumpacket** [TeamNewPipe/NewPipe#10411]
### Checklist

- [X] I am able to reproduce the bug with the latest version given here: [CLICK THIS LINK](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.25.2

### Steps to reproduce the bug

1. Go to https://www.youtube.com/watch?v=dLUoyW3l-D4

### Expected behavior

The video should play. I checked, and this is not an age-restricted video so https://github.com/TeamNewPipe/NewPipe/issues/10347 does not apply.

### Actual behavior

An exception occurs during loading of the video.

## Exception
* __User Action:__ requested stream
* __Request:__ https://www.youtube.com/watch?v=dLUoyW3l-D4
* __Content Country:__ US
* __Content Language:__ en-
* __App Language:__ en_US
* __Service:__ YouTube
* __Version:__ 0.25.2
* __OS:__ Linux Android 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
org.schabi.newpipe.extractor.stream.StreamInfo$StreamExtractException: Could not get any stream. See error variable to get further details.
	at org.schabi.newpipe.extractor.stream.StreamInfo.extractStreams(StreamInfo.java:188)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:82)
	at org.schabi.newpipe.extractor.stream.StreamInfo.getInfo(StreamInfo.java:73)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getStreamInfo$3(ExtractorHelper.java:119)
	at org.schabi.newpipe.util.ExtractorHelper.$r8$lambda$YTHJjScxCJNO1LTCqs3IKy35iyY(ExtractorHelper.java:0)
	at org.schabi.newpipe.util.ExtractorHelper$$ExternalSyntheticLambda3.call(R8$$SyntheticClass:0)
	at io.reactivex.rxjava3.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:43)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4855)
	at io.reactivex.rxjava3.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4855)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5377)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.rxjava3.core.Flowable.subscribe(Flowable.java:16013)
	at io.reactivex.rxjava3.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.rxjava3.core.Maybe.subscribe(Maybe.java:5377)
	at io.reactivex.rxjava3.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.rxjava3.core.Single.subscribe(Single.java:4855)
	at io.reactivex.rxjava3.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.rxjava3.core.Scheduler$DisposeTask.run(Scheduler.java:644)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:65)
	at io.reactivex.rxjava3.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:56)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)

```
</details>
<hr>

### Affected Android/Custom ROM version

LineageOS 18.1

### Affected device model

OnePlus 3T

### Additional information

This is happening across numerous videos, and from what I can tell it's restricted to specific channels, and only certain videos. The music channel the reported URL belongs to I can't play any recently uploaded videos from it, while other channels I can. This has been going on for about a week.

**Comment by AudricV** [comment]
Are you able to connect to `youtubei.googleapis.com`, the domain used to fetch `ANDROID` and `IOS` clients? Did you block this domain to avoid previous 403 HTTP issues on streaming URLs of the `ANDROID` client like written in the changelogs of 0.24.1 and 0.25.2?

If so, then #10347 and your issue have the same origin, the breakage of the extraction of the signature deobfuscation function; obfuscated signatures are mostly applied to musical videos on HTML5 clients (like `WEB`, i.e. the desktop YouTube website, and the client used to bypass age-restrictions).

**Comment by quantumpacket** [comment]
Hi, @AudricV I did discover that `youtubei.googleapis.com` was indeed blocked by NextDNS. Enabling it, the videos now load. I never would have thought that would have been the issue since it was only effecting some videos. I'm going to close this as a duplicate of #10347. Thank you for helping me in the right direction, and sorry for the dupe.
